.github/workflows/release.yml

```diff
@@ -47,12 +47,30 @@ jobs:
         with:
           path: target/release/${{ matrix.artifact_name }}
           name: ${{ matrix.target }}
+  build-linux-musl:
+    name: Build Static
+    runs-on: ubuntu-latest
+    env:
+      CARGO_TERM_COLOR: always
+      BINARY_NAME: rust-test1
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build-musl Linux x86
+      uses: shogan/rust-musl-action@master
```

Comment:
why induce another uses?

Reply:
not sure if dtolnay/rust-toolchain@stable support musl build...

I'm going to close this PR, and fallback to cmake-language-server...